mlir/tools/mlir-miopen-lib/mlir-miopen-lib.cpp

```diff
@@ -21,6 +20,8 @@
 #include <sstream>
 #include <string>
 
+using namespace mlir;
+
```

Comment:
Nit: Why adding this namespace? Looks like symbols below are scoped with the namespace under mlir/llvm already.

Reply:
Not all the symbols were so prefixed, and things broke without the `using namespace mlir;` that was in `LowerMIOpenOps.h`

Reply:
Okay gotcha, I think the reason why it wasn't an issue before is probably due to the mlir namespace spilled from the the header into other compilation units. Looks like your refactor corrected that.